source/people/main.rst

```diff
@@ -155,3 +157,4 @@ Alumni
    jacobson
    opotowsky
    ../jobs/programmer-2012
+   ../jobs/programmer-2010
```

Comment:
This was added accidentally as well... as part of the same change you already fixed